purchase_quick/models/product_product.py

```diff
@@ -0,0 +1,52 @@
+# © 2014 Today Akretion
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# @author Pierrick Brun <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    # for searching purpose
+    variant_specific_seller_ids = fields.One2many(
+        'product.supplierinfo', 'product_id')
+    po_line_ids = fields.One2many(
+        comodel_name='purchase.order.line', inverse_name='product_id',
+        help='Technical: used to compute quantities to purchase.',
+    )
+
+    @api.depends('po_line_ids')
+    def _compute_process_qty(self):
+        res = super(ProductProduct, self)._compute_process_qty()
+        if self.env.context['parent_model'] == 'purchase.order':
+            for product in self:
+                po_lines = self.env['purchase.order.line'].search([
```

Comment:
@PierrickBrun Could you maybe do the search outside the loop :
```suggestion
                po_lines = self.env['purchase.order.line'].search([('order_id', '=', self.env.context.get('parent_id'))])
                for product in self:
                     product_po_lines = po_lines.filtered(lambda l, p=product: l.product_id == p)
```